vanilla/scss/styles.scss

```diff
@@ -15,23 +15,74 @@ $grid-max-width: 92rem;
   color: $colors--light-theme--text-default;
 }
 
-//The following is a workaround as after spending a while exploring I can't work out how to escape the -- in the class name from the .rst file
-.p-notification-information {
-  @extend .p-notification--information;
+li > p {
```

Comment:
I'd also add a comment above this one explaining that Sphinx adds paragraphs inside list elements.